<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/imgs/images/2.jpg);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imgs/images/blue.jpg);
border:none;
border-radius: 15%;
color: white;
font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Already have an account?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imgs/images/blue.jpg);
border:none;
border-radius: 15%;
color: white;
font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imgs/images/white.jpg);
font: 18pt &quot;Segoe UI&quot;;
border:none;
color: black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imgs/images/white.jpg);
border:none;
color: black;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gpa">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imgs/images/white.jpg);
border:none;
color: black;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pass">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imgs/images/white.jpg);
border:none;
color: black;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_confirmPass">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>500</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/imgs/images/white.jpg);
border:none;
color: black;
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
